.github/workflows/docker-build.yml

```
@@ -0,0 +1,246 @@
+        uses: cloudposse/github-action-matrix-outputs-write@v1
+        with:
+          matrix-step-name: ${{ github.job }}
+          matrix-key: ${{ env.MATRIX_ID }}
+          outputs: |-
+            value: ${{ env.JSON_value }}
+            project: ${{ env.JSON_project }}
+            target: ${{ env.JSON_target }}
+            imageName: ${{ env.JSON_imageName }}
+            imageTag: ${{ env.JSON_imageTag }}
+
+  post-docker-build:
```

actionlint (1.7.4)

job "post-docker-build" needs job "prepare" which does not exist in this workflow

(job-needs)